docs/releases/release-1.0.md

```diff
@@ -0,0 +1,56 @@
+
+## Core features and APIs
+
+Core features and APIs are the fundamental functionalities of Kubeflow Pipelines. The most fundamental functionality is to manage the Kubeflow Pipelines resources (that is, pipeline, pipeline version, run, job, experiment) via the operations listed below. Active supports are available for core features and APIs.
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Pipeline         | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Pipeline Version | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Run              | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |        |         | - [x]     | - [x] |
+| Job              | - [x]  |  - [x] |- [x]|- [x] | - [x]  |         |           | - [x]  | - [x]   |
+| Experiment       | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+
+* Refer to the Kubeflow Pipelines [introduction]((https://www.kubeflow.org/docs/pipelines/overview/)) for
+  - An [overview](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) and [main concepts](https://www.kubeflow.org/docs/pipelines/overview/concepts/), for example, what is a pipeline or run in Kubeflow Pipelines.
+  - How to manage the Kubeflow Pipelines resources via the the [Kubeflow Pipelines UI](https://www.kubeflow.org/docs/pipelines/overview/interfaces/).
+* Refer to the Kubeflow Pipelines [SDK samples and tutorials](https://www.kubeflow.org/docs/pipelines/tutorials/sdk-examples/)
+for how to manage the Kubeflow Pipelines resources via the Kubeflow Pipelines client.
+* A complete API method list and a detailed description of parameters to those API methods are available at the Kubeflow Pipelines [API reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/).
+
+## Experimental features
+
+Experimental features are the enhanced functionalities available in Kubeflow Pipelines. They might be replaced or deprecated in the future due to the refactoring or redesign of Kubeflow Pipelines.
+
+* Visualization of output artifacts in two ways: [built-in visualization of selected artifact types](https://www.kubeflow.org/docs/pipelines/sdk/python-based-visualizations/) and [python based custom visualization](https://www.kubeflow.org/docs/pipelines/sdk/python-based-visualizations/).
+
+* [Building pipelines with the SDK](https://www.kubeflow.org/docs/pipelines/sdk/).
+
+* [Upgrade/Reinstall the Kubeflow Pipelines instance](https://www.kubeflow.org/docs/pipelines/upgrade/).
+
+* [Step caching](https://www.kubeflow.org/docs/pipelines/caching/).
+
+* Explore lineage or metadata of runs.
+
+* Separate the Kubeflow Pipelines resources with multiple user namespace support. [???]
```

Comment:
Can I have context info on the lineage explorer or the multi-tenant?

Reply:
I think this is the only public introduction for lineage explorer: https://cloud.google.com/blog/products/ai-machine-learning/introducing-cloud-ai-platform-pipelines.

For multi tenant, we can just mention it is not available in standalone and will be released with Kubeflow 1.1

I think you don't need to fill in the entire information for these. You can merge the PR first and I can follow up writing about these two areas.